libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m

```diff
@@ -935,20 +936,20 @@ + (NSString *)advancedAuthStateDesc:(SFOAuthAdvancedAuthState)authState
 
 - (NSURLSession*)session {
     if (_session == nil) {
-        _session = [NSURLSession sessionWithConfiguration:[NSURLSessionConfiguration ephemeralSessionConfiguration]];
+        SFNetwork *network = [[SFNetwork alloc] init];
+        _session = network.activeSession;
```

Comment:
To rephrase then: "Why?" 🙂  What's the value of using one instance of an `NSURLSession` over another?